dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:latest
+
+WORKDIR /usr/src/app/graphqlApp
```

Comment:
I'll take it, but I'd add to the backlog to fix the pathing to better reflect the app name